README.md

```diff
@@ -15,25 +15,53 @@
+## Installation
 
-The main goal of Langium is to lower the barrier of creating a DSL / low-code platform. We achieve this by providing a DSL that describes the syntax and structure of your language: the _grammar language_. Langium's out-of-the-box functionality is based on the information extracted from a grammar declaration. The Xtext framework has proved this approach to be suitable both for rapid prototyping and large-scale applications.
+Build your first language with Langium in our [online playground](https://langium.org/playground/).
 
-## How?
+Alternatively you can use yeoman to generate a [sample Langium project](https://langium.org/docs/getting-started/).
 
-Documentation is available at [langium.org](https://langium.org). Read the [getting started guide](https://langium.org/docs/getting-started/) for an introduction to language engineering with Langium.
+## Documentation
 
-Please read the [Contribution Guide](./CONTRIBUTING.md) if you want to contribute to Langium.
+You can find the Langium documentation on [the website](https://langium.org/).
+
+If you're new to building programming language, take a look at [our overview to see what Langium offers](https://langium.org/docs/langium-overview/). 
+
+The documentation is divided into several sections:
+
+* [Main Concepts](https://langium.org/docs/)
+* [Tutorials](https://langium.org/tutorials/)
+* [Advanced Guides](https://langium.org/guides/)
+* [Playground](https://langium.org/playground/)
+
+The documentation website is hosted in [this repository](https://github.com/langium/langium-website).
+
+## Examples
+
+We host a number of simple examples in our [main repo](https://github.com/langium/langium/tree/main/examples):
+
+* **[arithmetics](https://github.com/langium/langium/tree/main/examples/arithmetics)**: How to create an expression language + interpreter.
+* **[domainmodel](https://github.com/langium/langium/tree/main/examples/domainmodel)**: How to create a language with fully qualified name identifiers.
+* **[requirements](https://github.com/langium/langium/tree/main/examples/requirements)**: How to create a Langium project with multiple languages.
+* **[statemachine](https://github.com/langium/langium/tree/main/examples/statemachine)**: How to create a code generator.
+
+More complex examples are available as separate repositories in [our GitHub organization](https://github.com/langium):
+
+* **[lox](https://github.com/langium/langium-lox)**: Implementation of the Lox language from the popular book [Crafting Interpreters](https://craftinginterpreters.com/the-lox-language.html).
+* **[minilogo](https://github.com/langium/langium-minilogo)**: A language to draw logos. Shows how to integrate Langium in the browser.
```

Comment:
The language is not named like that because the purpose is to draw logos, but it's rather taken from the programming language named [Logo](https://el.media.mit.edu/logo-foundation/what_is_logo/logo_programming.html)